spec/index.bs

```diff
@@ -2487,6 +2503,12 @@ interface GPURenderPassEncoder {
     void setBlendColor(GPUColor color);
     void setStencilReference(GPUStencilValue reference);
 
+    void beginOcclusionQuery(GPUSize32 queryIndex, boolean preciseOcclusion);
```

Comment:
nit: preciseOcclusion = false

Reply:
Actually, I don't think we need this extra argument at all right now. We won't use it until we have an extension that needs it.

Reply:
hm... Never mind. I saw you added the extension string, but was thinking there was still more work to do on the API to add the extension (I didn't think about the fact that this is all there is).

Reply:
Done